package.json

```diff
@@ -354,6 +354,174 @@
 						}
 					}
 				}
+			},
+			{
+				"type": "reactnativedirect",
```

Comment:
Should we call this "reactnativehermes" instead of "direct" ? 'direct' implies that the previous debugging was 'in-direct'. I don't think that description would be accurate.

Reply:
@dhanvikapila technically, yes, previous debugging was "in-direct". The thing is that new debugger can be used to connect to the react native windows apps in future because old debugging method will be deprecated soon not only in vanilla RN, but in RNW, as well. Moreover, AFAIK RNW team at some point already support direct debugging. Then users may be confused, that they use "hermes" debugger for debugging RNW apps instead of "chakracore" debugger or something.

Reply:
Good point about RNW debugging.  I get why using "hermes" could be wrong in RNW scenario.   Let keep 'direct' for now.